pkg/api/customization/cluster/actions_etcd.go

```diff
@@ -106,8 +112,32 @@ func (a ActionHandler) RestoreFromEtcdBackupHandler(actionName string, action *t
 		}
 	}
 
+	if input.RestoreRkeConfig != "" && backup.Status.ClusterObject == "" {
```

Comment:
Is there a types PR to add the "RestoreRkeConfig" to input?

Reply:
yes already merged https://github.com/rancher/types/pull/1101

Reply:
Ah I see it in the PR desc, nevermind